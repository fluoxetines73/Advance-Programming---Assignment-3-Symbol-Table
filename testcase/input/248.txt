BEGIN
END
INSERT e_123_2r number
INSERT x number
ASSIGN x 789
RPRINT
BEGIN
END