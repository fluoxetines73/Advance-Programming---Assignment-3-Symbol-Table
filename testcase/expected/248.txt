success
success
success
x//0 e_123_2r//0